modules.d/35network-wicked/module-setup.sh

```diff
@@ -1,76 +1,155 @@
-        inst_dir /usr/lib/wicked/bin
-        inst_multiple "/usr/lib/wicked/bin/*"
-    elif [ -d /usr/libexec/wicked/bin ]; then
-        inst_dir /usr/libexec/wicked/bin
-        inst_multiple "/usr/libexec/wicked/bin/*"
-    fi
+    inst_dir /usr/lib/wicked/bin
     inst_dir /var/lib/wicked
 
-    inst_multiple "/etc/wicked/*.xml"
-    inst_multiple "/etc/wicked/extensions/*"
-    if [ -f /etc/dbus-1/system.d/org.opensuse.Network.conf ]; then
-        inst_multiple "/etc/dbus-1/system.d/org.opensuse.Network*"
-    elif [ -f /usr/share/dbus-1/system.d/org.opensuse.Network.conf ]; then
-        inst_multiple "/usr/share/dbus-1/system.d/org.opensuse.Network*"
-    fi
-    inst_multiple "/usr/share/wicked/schema/*"
-    inst_multiple "/usr/sbin/wicked*"
-
-    wicked_units=(
-        "$systemdsystemunitdir"/wickedd.service
-        "$systemdsystemunitdir"/wickedd-auto4.service
-        "$systemdsystemunitdir"/wickedd-dhcp4.service
-        "$systemdsystemunitdir"/wickedd-dhcp6.service
-        "$systemdsystemunitdir"/wickedd-nanny.service
-    )
-
-    inst_multiple "${wicked_units[@]}"
-
-    for unit in "${wicked_units[@]}"; do
-        sed -i 's/^After=.*/After=dbus.service/g' "$initdir/$unit"
-        sed -i 's/^Before=\(.*\)/Before=\1 dracut-pre-udev.service/g' "$initdir/$unit"
-        sed -i 's/^Wants=\(.*\)/Wants=\1 dbus.service/g' "$initdir/$unit"
-        # shellcheck disable=SC1004
-        sed -i -e \
-            '/^\[Unit\]/aDefaultDependencies=no\
-            Conflicts=shutdown.target\
-            Before=shutdown.target' \
-            "$initdir/$unit"
+    inst_multiple -o \
+        /usr/share/wicked/schema/addrconf.xml \
+        /usr/share/wicked/schema/bonding.xml \
+        /usr/share/wicked/schema/bridge.xml \
+        /usr/share/wicked/schema/constants.xml \
+        /usr/share/wicked/schema/dummy.xml \
+        /usr/share/wicked/schema/ethernet.xml \
+        /usr/share/wicked/schema/ethtool.xml \
+        /usr/share/wicked/schema/firewall.xml \
+        /usr/share/wicked/schema/gre.xml \
+        /usr/share/wicked/schema/infiniband.xml \
+        /usr/share/wicked/schema/interface.xml \
+        /usr/share/wicked/schema/ipip.xml \
+        /usr/share/wicked/schema/lldp.xml \
+        /usr/share/wicked/schema/macvlan.xml \
+        /usr/share/wicked/schema/modem.xml \
+        /usr/share/wicked/schema/openvpn.xml \
+        /usr/share/wicked/schema/ovs-bridge.xml \
+        /usr/share/wicked/schema/ppp.xml \
+        /usr/share/wicked/schema/protocol.xml \
+        /usr/share/wicked/schema/scripts.xml \
+        /usr/share/wicked/schema/sit.xml \
+        /usr/share/wicked/schema/team.xml \
+        /usr/share/wicked/schema/tuntap.xml \
+        /usr/share/wicked/schema/types.xml \
+        /usr/share/wicked/schema/vlan.xml \
+        /usr/share/wicked/schema/vxlan.xml \
+        /usr/share/wicked/schema/wicked.xml \
+        /usr/share/wicked/schema/wireless.xml \
+        "$dbussystem"/org.opensuse.Network.conf \
```

Comment:
In my previous PR, I didn't use `$dbussystem` because in SUSE it expands to /usr/share/dbus-1/system.d in dracut.sh, but wicked dbus configuration files can be located in /etc/dbus-1/system.d, depending on the SUSE distribution.